### PR TITLE
tests(e2e) skip flaky locality aware loadbalancing test

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
### Summary

I explained why we have to skip this in the test itself.

### Issues resolved

No issues.

### Documentation

- [X] No docs.

### Testing

- [ ] Unit tests
- [X] E2E tests
- [ ] Manual testing on Universal
- [ ] Manual testing on Kubernetes 

### Backwards compatibility

- [X] Add `backport-to-stable` label if the code is backwards compatible. Otherwise, list breaking changes.

### CODE REVIEW

File: test/e2e/trafficroute/universal_multizone/traffic_route.go [hunk 1]
```diff
@@ -369,7 +369,15 @@ conf:
 	})
 
 	Context("locality aware loadbalancing", func() {
-		It("should loadbalance all requests equally by default", func() {
+		// todo(jakubdyszkiewicz) test fails because of low --concurency in CI
+		// there are not enough "client -> zoneingress -> server" connections to have proper loadbalancing
+		// If a client is connected to ZoneIngress, ZoneIngress opens connection to a server.
+		// ZoneIngress is a simple TCP passthrough proxy therefore it cannot spread requests to many instances.
+		// If concurrency is low, we only iniate 2 connections to ZoneIngress which is not enough to cover all instances.
```
Comment (jpeach) [inline test/e2e/trafficroute/universal_multizone/traffic_route.go:376]:
```suggestion
// If concurrency is low, we only initiate 2 connections to ZoneIngress which is not enough to cover all instances.
```

**Review by jpeach — COMMENTED** [review]
IIUC, using `Pit` will make the test never run. What about conditionally skipping?

```Go
if runtime.NumCPU() < 4 {
        Skip("concurrency too low")
}
```

**Comment by lahabana** [comment]
ref #1920

**Review by bartsmykla — APPROVED** [review]
lgtm

**Pull request merged**